sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/CHANGELOG.md

```diff
@@ -63,24 +46,17 @@
 - Model Resource has a new parameter system_data
 - Model TrackedResource has a new parameter system_data
 - Model Cluster has a new parameter system_data
-
 ## 1.0.1 (2023-02-20)
-
 ### Other Changes
-
 - Upgraded dependent `Azure.Core` to `1.28.0`.
 - Upgraded dependent `Azure.ResourceManager` to `1.4.0`.
-
 ## 1.0.0 (2022-09-19)
-
 This release is the first stable release of the Redis Enterprise Management client library.
-
 ### Other Changes
-
 - Upgraded dependent `Azure.ResourceManager` to 1.3.1
 - Optimized the implementation of methods related to tag operations.
-
 ## 1.0.0-beta.2 (2022-08-29)
+- Exposed `JsonModelWriteCore` for model serialization procedure.
```

Comment:
this should be added under the latest version

Reply:
Fixed